### DISCUSSION TITLE
Get Report visualisation data

### DISCUSSION THREAD

**Discussion by willwainwright** [microsoft/Microsoft-Fabric-workload-development-sample#52] [Help] [answered]
Is there a way to extract data from a Power BI report visualisation and use as part of our workload object?

**Comment by omgharra** [comment, marked as answer]
Currently, there is no direct support or built-in feature that allows for the integration of workload objects with Power BI report visualizations. To use data from Power BI in other systems, you would need to explore indirect methods such as exporting data, using Power BI's REST API, or leveraging Power Automate for custom workflows.

**Reply by willwainwright** [reply]
Okay thanks. I saw there is a fabric API to get the report definition but that doesnt contain the queries to ge the data. So to clarify when interacting with other fabric workloads/experiences (e.g. Power BI); there are only the public APIS?

**Reply by omgharra** [reply]
Correct
